sdk/maps/azure-maps-geolocation/azure/maps/geolocation/_geolocation_client.py

```diff
@@ -0,0 +1,99 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+# pylint: disable=unused-import,ungrouped-imports, R0904, C0302
+from typing import Union, Any
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+
+from ._base_client import MapsGeolocationClientBase
+from .models import (
+    IpAddressToLocation
+)
+
+
+# By default, use the latest supported API version
+class MapsGeolocationClient(MapsGeolocationClientBase):
```

Comment:
To me, this seems like `GeolocationClient` would be more appropriate. @annatisch what do you think?